ppafm/cpp_utils.py

```diff
@@ -1,13 +1,33 @@
 import os
+import glob
+import platform
 
-recompile = True
-lib_ext   ='_lib.so'
+# Check for environment variable PPAFM_RECOMPILE to determine whether
+# we should recompile the C++ extensions.
+if 'PPAFM_RECOMPILE' in os.environ and os.environ['PPAFM_RECOMPILE'] != '':
+    recompile = True
+else:
+    recompile = False
+
+# Shared libraries are called .dll on Windows and .so on Linux by convention
+system = platform.system()
+if system == 'Windows':
+    lib_ext = '_lib.dll'
+else:
+    lib_ext = '_lib.so'
+
+cpp_modules = {
+    'PP': 'ProbeParticle',
+    'GU': 'GridUtils',
+    'fitting_': 'fitting',
+    'fitSpline_': 'fitSpline'
+}
 
 def work_dir( v__file__ ):
     return os.path.dirname( os.path.realpath( v__file__ ) )
 
 PACKAGE_PATH = work_dir( __file__ )
```

Comment:
```suggestion
PACKAGE_PATH = Path(__file__).resolve().parent
```
Same for all the other occurrences.

Reply:
Change pathlib done.